core/build.gradle

```diff
@@ -1,6 +1,15 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
 dependencies {
     implementation 'org.tomlj:tomlj:1.1.1'
 
+    implementation 'com.github.librepdf:openpdf:1.3.30'
+    implementation 'org.xhtmlrenderer:flying-saucer-core:9.1.22'
```

Comment:
Muzeme misto `flying-saucer` pouzit `'com.github.librepdf:openpdf-html:2.2.3'`? Vyzaduje to sice jeste `'org.slf4j:slf4j-api:2.0.16'`, ale prijde mi lepsi pouzivatnemichat knihovny, kdyz to neni treba